### DISCUSSION TITLE
"The puzzle of the galaxy with no dark matter"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#111] [Gravitation Models]
[https://www.iac.es/en/outreach/news/puzzle-galaxy-no-dark-matter](https://www.iac.es/en/outreach/news/puzzle-galaxy-no-dark-matter)

> This is the first time that a massive galaxy (it has a mass several times that of the Milky Way) does not show evidence for this invisible component of the universe. “This result does not fit in with the currently accepted cosmological models, which include dark matter”

**Comment by HanDeBruijn** [comment]
> If this the result, that NGC 1277 does not have dark matter, is confirmed, it would cast strong doubt on alternative models for dark matter, namely theories in which gravity is modified and the major part of the gravitational attraction within galaxies is due to a slight change in the law of gravity on large scales. "Although the dark matter in a specific galaxy can be lost, a modified law of gravity must be universal, it cannot have exceptions, so that a galaxy without dark matter is a refutation of this type of alternatives to dark matter" notes Trujillo.

Important conclusion is that **with NGC 1277 not only dark matter is rejected, but it is also a [MOND](https://en.wikipedia.org/wiki/Modified_Newtonian_dynamics) killer**.

Now some people might think that NGC 1277 is a killer of (our version of) the Variable Mass Theory as well, because of continuing advertizing with [MOND = UAC](https://github.com/orgs/a-cosmology-group/discussions/70#discussioncomment-6265312). But it's not what it seems to be. Confusion may arise because the title of the section is somewhat misleading; perhaps it would be better to replace it by $\mbox{MOND} \subset \mbox{UAC}$ (i.e. is a subset of).
The MOND = UAC equivalence is only valid for a special case, namely if the galaxy observed is one where new matter is created at the center. With other words: our galaxy must contain an **[Active galactic nucleus](https://en.wikipedia.org/wiki/Active_galactic_nucleus)**. The Wikipedia webpage about [Non-standard cosmology](https://en.wikipedia.org/wiki/Non-standard_cosmology#Redshift_periodicity_and_intrinsic_redshifts) says the following at the end of the linked section. _Halton Arp has proposed an explanation for his observations by a Machian "variable mass hypothesis". The variable-mass theory invokes **constant matter creation from active galactic nuclei**_. Arp's idea is precisely as copied and pasted into our theory.
> NGC 1277 is considered a prototype "relic galaxy" which means a galaxy which has had no interactions with its neighbours.

More important for us is the question: does that prototype "relic galaxy" contain an active galactic nucleus? If not, then why its description with MOND fails is readily explained. And indeed, in that case, our calculations confirm observations:
> The team discovered that the mass distribution in NGC 1277 was just the distribution of the stars.

![copy](https://github.com/a-cosmology-group/acg/assets/135433222/53b252c9-672b-4d97-aa05-cf41f32ce9f6)

It's interesting to make a word count of (supermassive) "black hole" occurrences in the accompanying paper, giving a total of **69** !!
I have never understood this. Even at first sight, it is obvious that galaxies have no sign of a black spot at their center. Rather the idea is that new matter emerges into our universe via active galactic nuclei, where Arp suggests there may be **white holes** rather than ([easy to fake](http://www.alternatievewiskunde.nl/melkweg/zwartgat.htm)) supermassive black holes.

**Comment by khuramonline** [comment]
This galaxy actually proves my point of view.

For galactic rotations, scientists wrongfully applied solar system specific Newton Theorem XXXI. This theorem was applicable to solar-planet systems or planet-moon systems **where larger chunk of total mass is located at the center**.

In normal galactic disks, larger chunk of mass is spread out throughout the disk. The applicable Newton Theorem was XXXIII which other scientists did not apply whereas MS Vera Rubin also had ended up only in a hybrid application of Theorems XXXI and XXXIII.

**Details in [in this portion of my book (2019)](https://zenodo.org/record/3530781):**

**Note:** My book (2019) failed to reach the correct position about MS Vera Rubin. **[So my updated stance about work of MS Vera Rubin is here](https://www.quora.com/Which-mistake-of-other-scientists-was-corrected-by-MS-Vera-Rubin-and-which-mistake-did-she-commit-in-her-own-work-that-eventually-prolonged-the-wrong-idea-of-dark-matter/answer/Khuram-Rafique).**

### Now about Galaxy NGC 1277:

NGC 1277 is a [Lenticular Galaxy](https://en.wikipedia.org/wiki/Lenticular_galaxy).

![image](https://github.com/a-cosmology-group/acg/assets/132253587/35505564-2a56-4b75-83ef-e03f152cf29f)

Now important and relevant point about Lenticular Galaxies is that they also have larger chunk of total mass concentrated in the center. **Therefore MS Vera Rubin type of hybrid application of Theorems XXXI and XXXIII is likely to give the result of dark matter free galaxy.**

Following screenshot shows that greater chunk of mass is located at the center in such galaxies:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/c8b79b0f-4c7d-4d00-8ffc-3b7fda471fc6)

**Reply by khuramonline** [reply]
Moreover, following are highlights about the huge size of black hole located at the center of NGC 1277:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/08de867b-7872-4109-a998-32c7f7ed90c9)

So on lower estimate scale, mass of black hole of NGC 1277 is 2 to 5 BILLION Solar masses. As a comparison to this, the black hole of our own galaxy has mass of only  4 MILLION Solar masses:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/7dc511ea-e098-4ff5-93d3-50c3da91d3a9)

That means central black hole for NGC 1277 is not only much massive but the central bulge is also much greater. **These two things combined justify the hybrid application of Theorems XXXI and XXXIII that will give the result of dark matter free galaxy.**

**Reply by HanDeBruijn** [reply]
>PROPOSITION LXXI. THEOREM XXXI.
_The same things supposed as above, I say, that a corpuscle placed without the spherical superficies is attracted towards the centre of the sphere with a force reciprocally proportional to the square of its distance from that centre._

In modern parlance that is

$$
F = G\frac{Mm}{r^2}
$$

Where $F=$ _a force_, $M=$ mass of _the sphere_, $m=$ mass of _a corpuscle_, $r=$ _distance from that centre_ (and $G=$ gravitational constant).

>PROPOSITION LXXIII. THEOREM XXXIII.
_If to the several points of a given sphere there tend equal centripetal forces decreasing in a duplicate ratio of the distances from the points; I say, that a corpuscle placed within the sphere is attracted by a force proportional to its distance from the centre._

A modern formulation is found in [Shell theorem](https://en.wikipedia.org/wiki/Shell_theorem) (Wikipedia):
>A corollary is that inside a solid sphere of constant density, the gravitational force within the object varies linearly with distance from the center.

Formally:

$$
F = G\frac{Mm}{R^2}\frac{r}{R}
$$

Where $R=$ radius of the sphere, rest of the symbols as before.

If we equate the forces above with the _centripetal forces_ at the _corpuscle_, then with the first force the classical Keppler laws for orbital velocities are recovered. With the last force one obtains, for a circular orbit in the galaxy:

$$
G\frac{Mm}{R^2}\frac{r}{R} = m\omega^2 r \quad \Longrightarrow \quad \omega = \sqrt{G\frac{M}{R^3}}
$$

Where $\omega=$ angular velocity. This would mean that velocities are like in a [Flywheel](https://en.wikipedia.org/wiki/Flywheel), but **a galaxy is not a flywheel** either !!

>**hybrid application of Theorems XXXI and XXXIII is likely to give the result of dark matter free galaxy.**

Any [decent mathematical model](https://arxiv.org/pdf/1212.5317.pdf) of a galaxy ***is*** an implementation of "XXXI and XXXIII". Astronomers may be wrong but they are not stupid. Yet they find that the observed flat rotation curves are **not** reproduced by their calculations.

>**the hybrid application of Theorems XXXI and XXXIII that will give the result of dark matter free galaxy.**

~~Nope.~~

**Reply by HanDeBruijn** [reply]
>### Now about Galaxy NGC 1277:
>NGC 1277 is a [Lenticular galaxy](https://en.wikipedia.org/wiki/Lenticular_galaxy).

Quote from Wikipedia:
>Lenticular galaxies [ .. ] have very little ongoing star formation.

Which is essential for [MOND = UAC](http://www.alternatievewiskunde.nl/cosmology/vervolg.htm): little or no creation of new mass makes that MOND does not apply.

>### Supermassive black hole

That's a simple one: **Black Holes do not exist**. Period.

**Comment by khuramonline** [comment]
MS Vera Rubin did reach to the hybrid application of Theorems XXXI and XXXIII. She calculated mass up to each concentric shell and duly ignored outer mass for that r in terms of Theorem XXXIII.

But she stayed with Theorem XXXI as she applied inverse square distance from center law.

Full application of Theorem XXXIII would have required inverse linear distance from center law.

So above was the hybrid application of both Theorems by MS Vera Rubin.

**And yes ... actually a hybrid application was required for galactic disk.**

**But that Hybrid had to be different sort of application.**

Yes ... inverse square law was not applicable. But simple inverse linear law was also not applicable.

For a real applicable Hybrid application, it was required to work out an in-between of inverse square and inverse linear law.

I wrote in my book (2019) that within the bulge area of disk, mass density remains the same. And for outside of bulge area up to the edge of disk, mass remains the same.

At that time (2019) I did not know that MS Vera Rubin had also stated both above points in her 1983 work.

Now exact linear distance law is applicable if density is same.

And in case mass is same with reduced density across each concentric shell, then an in-between law was required to be worked out. 

If we apply same Hybrid application as that of MS Vera Rubin, then I restate that it is likely that galaxy NGC 1277 would be found out to be dark matter free. Due to the reason that a large chunk of total mass of this galaxy is located at central bulge.

**Reply by khuramonline** [reply]
And for a better hybrid application that has duly worked out the in-between law, all the normal galaxies would be dark matter free. 

One point may remain outstanding that I may mention at suitable time.

**Reply by HanDeBruijn** [reply]
>For a real applicable Hybrid application, it was required to work out an in-between of inverse square and ~~inverse~~ linear law.

>And for a better hybrid application that has duly worked out the in-between law, all the normal galaxies would be dark matter free.

With Theorem XXXI we have, with meaning of the symbols the same as in my previous postings and $v=$ speed of the _corpuscle_:

$$
\frac{GM}{r^2} = \frac{v_1^2}{r} \quad \Longrightarrow \quad v_1 = \sqrt{\frac{GM}{R}}\left(\frac{r}{R}\right)^{-1/2}
$$

With Theorem XXXIII we have, also with meaning of the symbols the same:

$$
\frac{GM}{R^2}\frac{r}{R} = \frac{v_2^2}{r} \quad \Longrightarrow \quad v_2 = \sqrt{\frac{GM}{R}}\left(\frac{r}{R}\right)
$$

How do we make a **Hybrid** of the two outcomes $v_1$ and $v_2$. It's not unreasonable to take the [Geometric mean](https://en.wikipedia.org/wiki/Geometric_mean) of the two velocity profiles, giving $v=\sqrt{v_1 v_2}$ or

$$
v = \sqrt{\frac{GM}{R}}\left(\frac{r}{R}\right)^{1/4}
$$

Contrary to what I expected, the result is not so bad, if applied to _all the normal galaxies_.

![kuram](https://github.com/a-cosmology-group/acg/assets/135433222/392a5a6f-19f0-4f63-8dba-8dcce0014c4c)

**Comment by khuramonline** [comment]
Thanks for taking pain to formulate it and getting the results. On technical maths I cannot comment as it is the domain of mathematicians to check it.

However, I think for the term 'hybrid' it may be just convenient to rely on geometric mean ... But it all would actually depend on density profile.

Same density (as of bulge area) means straight inverse linear law.

Lower density per concentric shell towards the outer edges ... the hybrid law is in-between of linear and inverse square law --- with more inclination towards linear... however it all would depend on actual density profile.

**Comment by budrap00** [comment]
Bravo Gentlemen, you've got the right idea. It's a geometry problem - how to transition from the spherical inverse square drop-off to the inverse linear relation of the disk. MOND manages the problem by ignoring the geometry and heuristically deriving a relation in terms of the acceleration scale. Nice enough as math goes, but it does not offer anything like a satisfactory account of the cause of that relation.

The trick to making your approach work generally for all disk galaxies would be in making the mass distribution of the disk predictable from the overall properties of the galaxy. For instance if it is assumed that galaxies form in the Arpian manner, then there could well be a dynamical relationship between the overall mass of the galaxy and the mass of the spherical core, that dynamically determines the mass distribution of the disk during galaxy formation as the galaxy spins-out the disk.

**Reply by HanDeBruijn** [reply]
@budrap00 @khuramonline . Would you both please notice that it is  _the ~~inverse~~ **linear** relation of the disk_.

**Reply by khuramonline** [reply]
Yes I did notice. And did reach to the conclusion that it is actually simple inverse linear relation.

Now if I read my book, I notice various mistakes but cannot do anything to correct those mistakes.

There are also same kind of mistakes in Principia (in translation at least) ...

![image](https://github.com/a-cosmology-group/acg/assets/132253587/4545f841-114a-4b7d-a768-b415cafe0f7e)
 
Here "inverse" is missing. And "inverse" is required.

**Reply by HanDeBruijn** [reply]
No, **"inverse" is wrong** and Principia is alright !!!

**Reply by khuramonline** [reply]
Yes you are right. It is talking only about linear relation. small distance small force, greater distance, greater force.

In my book, I also seem to be perplexed. for example I used following diagram:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/9a3495da-2130-432f-8a9f-84b8874cd5ce)

But I admit, overall my concept was not clear. Later in the book I did use the term 'inverse'. 

But I used the correct underlying concept ... that gravitational force linearly increases with distance.

Following is example from book:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/fe314c12-0b4d-4d57-aeca-a3397700f19a)

See that basic principle that less distance (from center), less gravity is correct in basic analysis of book.

same is the treatment in the rest of the analysis of book. However, I do admit that in context of XXXIII, wherever I used term 'inverse' that is wrong.

Secondly I formulated nothing in this section. I proceeded only with concepts. Concept was correct that less distance, less gravity. More distance, more gravity. And it varies linearly with the distance.

Thanks for highlighting this point ... I will seriously think about how to rectify this problem for an already published book.

**Reply by khuramonline** [reply]
Now concept of Hybrid Application of Theorems XXXI and XXXIII needs a review.

By the time of writing the book (2019) I did not know that MS Vera Rubin had already achieved a basic form of hybrid application. I had read her 1983 paper afterwards and it was through 1983 paper I realized that she had actually applied a basic form of hybrid application.

By the time of writing book (2019) I "knew" in a perplexing way that actual applicable theorem should be between "inverse square distance" and "inverse linear distance".

But somehow I forgot to mention this point in the book. So the book remained totally silent about any form of hybrid application.

And now it is the time to get clear on the issue of Hybrid application.

 Following is from Wikipedia article (shell theorem):

![image](https://github.com/a-cosmology-group/acg/assets/132253587/7e7bc9ae-c48e-4dc0-a077-6415b396fe53)

So linear distance proportionality law is for constant density and the gravitational mass is proportional to distance cube.

Mass density is constant only for bulge area ... For area outside bulge throughout to the edge of disk, mass is constant and density is reducing layer upon layer. (i.e. layer means "concentric shell").

Hence, for this area, mass is proportional to a quantity lesser than distance cube.

And to find out the hybrid law, a quantity lesser than distance cube will be divided by distance square.... 

The answer will be less than r.

Somehow, gravity still be proportional to distance -- however for every r, there will be even greater increase in gravity... That "even greater" will however be far less than square of distance. **Can be regarded as slightly greater than linear.**

My mistakes are not relevant ... Mistake of official theory becomes more apparent through my mistakes.

Since I am not an (official) scientist so I do humbly accept my mistakes ... even if in this post if any expert would point out the mistake, I will accept. Task is to sort out, dig out or figure out the matter.

**Comment by budrap00** [comment]
`Any [decent mathematical model](https://arxiv.org/pdf/1212.5317.pdf) of a galaxy is an implementation of "XXXI and XXXIII". Astronomers may be wrong but they are not stupid. Yet they find that the observed flat rotation curves are not reproduced by their calculations.`

Which simply means that Newtonian Dynamics are not applicable on galactic scales - that's old news. What happened after that discovery was, of course, that no serious effort was given to devising better mathematics that would be more appropriate for a large distributed mass system. Instead the problem was off-loaded onto physical reality which was deemed to be hiding just enough mass to compensate for ND's failure. The rest as they say is history, and it ain't pretty.

A disk galaxy can be thought of as having two distinct geometries, the roughly spherical geometry of the central core, and the roughly circular geometry of the "flat" disk. The gravitational intensity drops off as 1/r^2 for any radial distance beyond the surface of a sphere, while the gravitational intensity should decline as 1/r radially outward from beyond the outer edge of the disk mass. How gravity functions between the outer edge of the disk and the surface of the central core is what we don't have a good qualitative or quantitative model for - any center of mass based calculations will almost certainly be wrong because they would grossly misrepresent the actual mass distribution.

It is unfortunate that Zwicky's early concept of gravitational viscosity was set aside with a spurious argument from a mathematical analysis of globular clusters. Gravitational viscosity would seem to be a promising qualitative approach to the disk problem but I don't think there are any existing math formalisms that can be readily fit to that analysis. Any math model that actually matters will have to reflect the physics of the disk geometry in terms of the mass distribution. Center of mass calculations are inappropriate for large distributed mass systems like galactic disks.

Fluid dynamics might be a fruitful starting point for galactic disk analysis but the mass dependency of gravitational behavior vs charge dependency of fluids means the two types of viscosity would not have identical characteristics and would therefore require somewhat different mathematical representations. ND does not work out of the box for galactic disks and GR is so mathematically complex that while it may be theoretically possible for that purpose, it is simply unworkable in practice. Maybe Einstein's GR needs a modern day Oliver Heaviside to make it tractable.

**Reply by RedshiftDrift** [reply]
> How gravity functions between the outer edge of the disk and the surface of the central core is what we don't have a good qualitative or quantitative model for

Sure we do! The numerical calculation in [arXiv:1210.1998](https://arxiv.org/abs/1210.1998) gives, for a central core with spherical geometry and a disk with circular geometry, the rotation curves everywhere from the center to the edge of the galaxy.  The disk density is modeled with an [exponential profile](https://en.wikipedia.org/wiki/Galactic_disc#Radial_profile) that matches observations.

Caveat: *The explanation for the missing mass given in that paper is now known to be incorrect.*

**Reply by khuramonline** [reply]
In my opinion --- **there is NO experimental low mass to luminosity ratio**.

Within the disk area from the edge of bulge towards edge of the disk, mass is (almost) same for each concentric shell (complete circumference of layers).

**Whereas experiments never determined the luminosity for each complete concentric shell (complete cicumfrence of the layer).**

I have explained in my book (2019) -- following is the screenshot:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/af00017d-5927-42fd-91fd-dd8812458baf)

**Scientists actually committed the mistake of comparing mass of total concentric shells with luminosity of only the radial lines.**

Following is from MS Vera Rubin (1983):

MS vera Rubin (1983) clearly shows that she had accurately realized that within the galactic disc, the mass in every concentric shell (layer) remains the same. This was right treatment.

What she missed was that **she missed to apply the same concept of quantity per concentric shell to luminosity**. In my book, I pointed out that luminosity does not decrease over successive outer layers within the galactic disc. Because luminosity only decreases on radial distance from the center. But for the complete outer layer, luminosity remains the same.

MS Vera Rubin clearly missed this point as highlighted in following screenshots:

Following is screenshot from MS Vera Rubin (1983)

![image](https://github.com/a-cosmology-group/acg/assets/132253587/f1752fec-4233-475b-ba0e-c66c51840102)

And following is from same para (3)… **This is the point where MS Vera Rubin missed to apply the concept of luminosity of each concentric shell. If she was talking about mass of each concentric shell then she should have talked about luminosity also for each concentric shell and not in terms of radial distance from center:**

![image](https://github.com/a-cosmology-group/acg/assets/132253587/307f6320-30c4-4e88-8d13-840db08772e2)

And following screenshot contains the title of paper:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/2ca20337-9c9d-4675-aedd-97a847f7bc76)

**Reply by HanDeBruijn** [reply]
@budrap00 writes:
>Maybe Einstein's GR needs a modern day Oliver Heaviside to make it tractable.

Maybe the world only needs Another Cosmology Group, to make such a _modern day Oliver Heaviside_ **observable**.

**Comment by khuramonline** [comment]
More about the topic of hybrid application of Theorems XXXI and XXXIII:

A further review of this topic was needed because XXXI is about INVERSE square distance law whereas XXXIII is about DIRECT linear distance law. Here it is the contrast of INVERSE/DIRECT which is the reason for such a further review.

So, my updated stance about this issue is that Hybrid Application is of only one type that was adopted by MS Vera Rubin.

It is to basically apply (solar system/ planet-moons system specific) XXXI with inverse square distance law and take only one component of XXXIII that is to take matter quantity only up to radius r.

That is -- to ignore the matter outside of r.

If there is one big planet and one smaller moon, this hybrid application will be perfect. For solar system having central mass more than 99%, again this system will work, with only minor deficiency.

And for galactic disks, this system will only work for galaxies of type NGC 1277, with minor deficiencies.

For all the other normal galaxies, this application will give the need for dark matter.

Now if we want to alter the square distance law ... then it is basically to switch from inverse application to direct application.

Therefore, any such attempt should not be regarded as hybrid application of XXXI and XXXIII.

If we are taking matter quantity only up to radius r and adopting a variation of **"direct linear relation with distance r"** such as a **"direct relation with a quantity slightly less than r"** then it is NOT the case of Hybrid application of XXXI and XXXIII.

Then it is the case of MODIFICATION of XXXIII.

And such a modified theorem will work fine for normal disk galaxies and no need of dark matter will arise.

For galaxies like NGC 1277, such a modified theorem will not work and there will be considerable discrepancy between calculated rotation and observed rotation.

**Comment by khuramonline** [comment]
On his [blog](https://tritonstation.com), Mr. Stacy McGaugh has presented a case of NGC 1277 in favor of MOND. His point is that acceleration (of rotational contents) in this galaxy is not low enough to touch or reach to the MOND (low acceleration) regime.

This is valid argument in favor MOND as long as the only contender is the dark matter regime.

**A Note:** Dark matter regime also has an excuse that dark matter theory is not violated by the mere fact that any particular part of universe (e.g. any individual galaxy etc.) is devoid of dark matter.

Here this excuse of the dark matter regime goes on weak footings because in NGC 1277 (according to the point of view of MOND) the absence of dark matter has a reason that rotation acceleration is not too low in this galaxy. Therefore, it is **not just by chance** that this galaxy has no dark matter. Something measurable is linked with the fact that this galaxy has no dark matter.

Now Stacy McGaugh did say that this galaxy has high rotation speed and he also discusses supermassive black hole of this galaxy.

Here comes the point of view of Theorem XXXIII.

If there is high ratio of central mass within the galaxy then it is NOT a proper case of XXXIII. For such a galaxy, a scenario similar to solar system will hold. For such a galaxy, inner most stuff should have high rotation speed and outer stuff should have lower rotation speed.

That also simply means --- No dark matter.

MOND does say that rotation acceleration is not such low and thus not below the limit of MOND for galaxy NGC 1277. MOND also discusses central high mass. But MOND does not attempt to link high rotation with high central mass.

High rotation is simply due to high central mass ratio. When there is a great central mass then more appropriate theorem is XXXI in such hybrid form with XXXIII as was adopted by MS Vera Rubin, and the result will be dark matter free galaxy with minor discrepancies.

If contenders are only dark matter and MOND, then MOND wins in this case.

But if there is another contender i.e. Theorem XXXIII, then victory would go to Theorem XXXIII.

Why?

Because MOND exists on account of the argument that Newton's Theory fails at flat rotation curves of galaxies.

But if Newton's theory does not actually fail by virtue of Theorem XXXIII **then argument of the MOND fails.**

Secondly, any amendment in Theorem XXXIII is not a fundamental amendment of the theory. Theory remains the same only details of implementation via a new modified theorem comes in.

Whereas MOND is the name of a fundamental change of the theory. And question of any fundamental change of the theory should arise only if main theory fundamentally fails.

Therefore if Theorem XXXIII is the third contender, then victory goes to Theorem XXXIII.

**Comment by khuramonline** [comment]
There is an outstanding question from Mathematicians.

It is about formulation of gravity formula "M x m" divided by distance square.

This has been outstanding question for me for many years. Now I think I can confidently say that Newton never meant "M x m".

He was only talking about "M + m".

He never formulated this formula. He has only provided few logical propositions where he did not use terms multiplication or addition. He used the term "several" (at least as per English translation). Following are screenshot:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/856359a9-f3f5-4675-9d29-74bf8f3e3bc3)

![image](https://github.com/a-cosmology-group/acg/assets/132253587/32fcc853-34f6-4cda-953b-f0024d5f4375)

Early on, I did try to conclusively determine the meaning of "several" but could not reach to conclusive point.

Now a review of Theorem XXXIII helped me to say with enough confidence that Newton is not talking about multiplication. He is talking about addition.

In previous comment I had used a screenshot of Wikipedia article about Shell Theorem. I copy it here again:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/a90ab66b-024c-43c0-9820-1b6dd0da3be6)

So for a constant density, the total matter within a given radius (r) is proportional to "r cube". 

This formula will give just the total mass. And for the purpose of the determination of gravity, only this total mass will be used.

Value of G (official) is just a worked back value that works with observations.

Theorem XXXI also works only with total available mass and that mass is not multiplied by itself or any other mass for the determination of gravity.

For me this may be resolved. But this should be at least an outstanding question for mathematicians.

**Reply by Francois-Zinserling** [reply]
@khuramonline the equation was credited to Newton but was not entirely derived by him. It is even alleged that he had the 1/r^2 wrong (he thought 1/r) and was corrected by Hooke.
Even when the equation was formulated, it took years before they could get a value for G, because the earth's mass was not known at the time.
People get the idea that immediately after Principia people launched off (pun intended) and did gravitational mathematics. I think it took many years before the equation became useful.


As for M*m, it comes from the following relation:
Big Mass M exerts a force on small mass m. The outgoing field is proportional to its own mass M and the effect on subject is dependent on the subject's mass m,  so the "force" is proportional to m*M
AND mass m exerts a force on mass M also. ,  so it is proportional to M*m, which of course is the same as m*M above, but this must not be ignored, even though it is exactly equal.

The equation F = GMm/r^2 is the sum of the 2 above forces, (even though they are equal) because both contribute.  So one day when we find out what G is, it will be (something) divided by 2.

**Reply by khuramonline** [reply]
My point is that "equation" was not even "derived" by him. He just presented logical propositions and he used term "several". I tried to check the meaning of term "several" within mathematical context and it was not clear however its meaning inclined more with addition and not multiplication. But that was not a conclusive point.

Newton might have learned distance square from Robert Hook. But he wrote Principia after having learned it.

Even if we multiply M by M in any theorem, we can always work out value of G that can work with observations.

But in Theorems that I have pointed out, just the total mass has been used. Especially Theorem XXXIII where for a given radius, mass (due to having constant density), total mass (for the purpose of gravity) is given by "r cube" for a given radius "r".

I hope that another working value of G also can be worked out that will work with "M + m".

**Comment by khuramonline** [comment]
Have found another relevant point from Principia.

It is about **inverse linear gravity** from **surface of planet to the center**.

![image](https://github.com/a-cosmology-group/acg/assets/132253587/4fa6d62e-568e-4986-b528-d0203b20af3c)

That mean **from center to (outer) surface**, gravity is **direct linear**.

**Reply by khuramonline** [reply]
^^^
Here this Prop. LXXIII Book I is referring to same Theorem XXXIII as clear in following screenshot:

![image](https://github.com/a-cosmology-group/acg/assets/132253587/84c0a2ef-be5b-4eb8-91fb-08133d101368)

And now this Prop. IX further makes it clear that that any error in the inverse linear (from surface to center) will be due to inequality of the density involved. That means, Newton had acknowledged that modifications in Theorem XXXIII will be required if density is not the same (per concentric shells).

Now only limitation remains that Theorem XXXIII is for sphere and not for disk.

The answer to this problem is that Official Theory has already applied Theorem XXXI (with minor modification) to disk whereas XXXI is also basically for sphere. For this, official theory has the reason that galactic disks are symmetrical therefore same sphere related gravity rules apply to disk.

However, to be precise, a disk related Theorem or Mathematical Proof may not so far be available which is possible to be developed by any genius mathematician.
